src/features/modelsCreator.js

```diff
@@ -6,7 +6,13 @@ const { REACT_APP_BASE_URL } = process.env;
 
 export const getCarModels = createAsyncThunk('model/allCarModels', async ({ carBrandId }) => {
   return await axios
-    .get(`${REACT_APP_BASE_URL}/models`, { params: { carBrandId } })
+    .get(`${REACT_APP_BASE_URL}/models`, {
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
+        Authorization: `JWT ${localStorage.getItem('token')}`
+      },
```

Comment:
duplication of headers code, try to make this for all axios requests https://github.com/axios/axios#interceptors